hack/minikube-rook-setup.sh

```diff
@@ -127,9 +128,13 @@ then
         ## Install rook-ceph ##
         kubectl apply -f "${ROOK_SRC}/common.yaml" --context="${PROFILE}"
         kubectl apply -f "${ROOK_SRC}/crds.yaml" --context="${PROFILE}"
-        # Fetch operator.yaml and enable CSI volume replication
-        kubectl apply -f https://raw.githubusercontent.com/csi-addons/volume-replication-operator/v0.1.0/config/crd/bases/replication.storage.openshift.io_volumereplications.yaml --context="${PROFILE}"
-        kubectl apply -f https://raw.githubusercontent.com/csi-addons/volume-replication-operator/v0.1.0/config/crd/bases/replication.storage.openshift.io_volumereplicationclasses.yaml --context="${PROFILE}"
+
+        # Enable CSI addons (for volume replication) (using main till a release is available with lastSyncTime)
+        kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/main/deploy/controller/crds.yaml
+        kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/main/deploy/controller/rbac.yaml
+        kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/main/deploy/controller/setup-controller.yaml
```

Comment:
Why not apply? With this the script is not idempotent.

We can extract the base url to a variable for readability:

    CSI_ADDON_SRC="https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/main/deploy/controller"
    ...
    kubectl apply -f ${CSI_ADDON_SRC}/...

When we move to release only the variable will have to change.

Reply:
Addressed